frontend/src/app/offers/create-offer/create-offer.component.ts

```diff
@@ -0,0 +1,196 @@
+import 'rxjs/add/operator/filter';
+import 'rxjs/add/operator/do';
+import { Component, OnDestroy, OnInit, ViewChild } from '@angular/core';
+import { AppOffer, ApiOffer, BaseOffer } from 'app/homepage-offer/offers.model';
+import { AuthService } from 'app/auth.service';
+import { User } from 'app/user';
+import { Observable } from 'rxjs/Observable';
+import { OffersService } from 'app/homepage-offer/offers.service';
+import { ActivatedRoute, Params, Router } from '@angular/router';
+import { environment } from 'environments/environment';
+import { Subscription } from 'rxjs/Subscription';
+import { FileReaderEvent, FileReaderEventTarget } from '../../models';
+import { NgForm, FormGroup } from '@angular/forms';
+import { FormBuilder } from '@angular/forms';
+import { Validators } from '@angular/forms';
+import { Image } from 'app/homepage-offer/image.model';
+import { Response } from '@angular/http';
+import { HttpClient } from '@angular/common/http';
+@Component({
```

Comment:
Add the empty line after imports - let's keep the style from the rest of the app